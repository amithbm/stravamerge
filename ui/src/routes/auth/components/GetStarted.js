import React from 'react'
import { Panel,Grid,Row,Col } from 'react-bootstrap'
import SignInButton from './SignInButton'
import SignInGuestContainer from '../containers/SignInGuestContainer'
import wheel from '../../../images/wheel.jpg'

export default () => {
    return  <Grid fluid style={{ backgroundImage: "url('" + wheel + "')", backgroundSize: "cover" }} className="seamless-banner"> 
                <Grid>
                    <Row className="m-t-2 m-b-2">
                        <Col xs={3}></Col>
                        <Col xs={6}>
                            <Panel>
                                <div className="m-b-2">To analyse your rides please sign in to Strava.</div>
                                <SignInButton className="m-t-2" />
                                <SignInGuestContainer  />
                                <div className="m-t-2"><em>If you don't have a Strava account and want to see a demo just use the guest access button - you won't be asked to create an account or enter any personal details.</em></div>
                            </Panel>
                        </Col>
                        <Col xs={6}></Col>
                    </Row> 
                </Grid>
            </Grid>
}