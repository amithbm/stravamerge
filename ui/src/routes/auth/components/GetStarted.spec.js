import React from 'react'
import {shallow} from 'enzyme'
import GetStarted from './GetStarted'
import toJson from 'enzyme-to-json'

describe('components', () => {
    describe('<GetStarted />', () => {
        it('renders as expected', () => {
            const component =  shallow(<GetStarted />)
            const tree = toJson(component)
            expect(tree).toMatchSnapshot()
        })
    })
})