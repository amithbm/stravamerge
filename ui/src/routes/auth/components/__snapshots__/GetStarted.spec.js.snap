exports[`components <GetStarted /> renders as expected 1`] = `
<Grid
  bsClass="container"
  className="seamless-banner"
  componentClass="div"
  fluid={true}
  style={
    Object {
      "backgroundImage": "url(\'wheel.jpg\')",
      "backgroundSize": "cover",
    }
  }>
  <Grid
    bsClass="container"
    componentClass="div"
    fluid={false}>
    <Row
      bsClass="row"
      className="m-t-2 m-b-2"
      componentClass="div">
      <Col
        bsClass="col"
        componentClass="div"
        xs={3} />
      <Col
        bsClass="col"
        componentClass="div"
        xs={6}>
        <Panel
          bsClass="panel"
          bsStyle="default"
          defaultExpanded={false}>
          <div
            className="m-b-2">
            To analyse your rides please sign in to Strava.
          </div>
          <Component
            className="m-t-2" />
          <Connect(withRouter(Component)) />
          <div
            className="m-t-2">
            <em>
              If you don\'t have a Strava account and want to see a demo just use the guest access button - you won\'t be asked to create an account or enter any personal details.
            </em>
          </div>
        </Panel>
      </Col>
      <Col
        bsClass="col"
        componentClass="div"
        xs={6} />
    </Row>
  </Grid>
</Grid>
`;
